### PR TITLE
Update Get-PnPWebPart.md

### PR DESCRIPTION
**Pull request by russgove**
only returns webparts from classic pages

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPWebPart.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ online version: https://pnp.github.io/powershell/cmdlets/Get-PnPWebPart.html
 
 > [!TIP]
 > We encourage you to make improvements to this documentation. Please navigate to https://github.com/pnp/powershell/blob/dev/documentation/Get-PnPWebPart.md to change this file.
+> [!TIP]
+> This command only returns webparts from Classic Pages
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPWebPart.md:17]:
```suggestion
>
> This command only returns webparts from Classic Pages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @russgove, kindly commit the suggested changes. Many thanks.

File: sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPWebPart.md (hunk 1 above)
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPWebPart.md:17]:
```suggestion
> This command only returns webparts from classic pages.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by get-itips** [comment]
how are you?

Even though the PnP documentation articles are on this repository, I need to kindly redirect you to open PnP cmdlets issues/pull requests [here](https://github.com/PnP/PowerShell/), in that repository, where a member of the PnP community will be able to correct the article/review your PR with your feedback.

**The documentation for PnP cmdlets is then synced from that repo into this one.**

Thank you!

cc @yogkumgit

**Comment by tseward** [comment]
Closing issue as this change needs to be proposed elsewhere. @yogkumgit

**Pull request closed without merging**